bindings/pyfiction/include/pyfiction/pybind11_mkdoc_docstrings.hpp

```diff
@@ -0,0 +1,15036 @@
+a given one. Thereby, only cardinal directions are being considered,
+i.e., the container contains all coordinates ``ac`` for which
+``is_adjacent(c, ac)`` returns ``true``.
+
+Coordinates that are outside of the layout bounds are not considered.
+Thereby, the size of the returned container is at most 4.
+
+Parameter ``c``:
+    Coordinate whose adjacent ones are desired.
+
+Returns:
+    A container that contains all of ``c``'s adjacent coordinates.)doc";
+
+static const char* __doc_fiction_cartesian_layout_adjacent_opposite_coordinates =
+    R"doc(Returns a container that contains all coordinates pairs of opposing
+adjacent coordinates with respect to a given one. In this Cartesian
+layout, the container will contain (``north(c)``, ``south(c)``) and
+(``east(c)``, ``west(c)``).
+
+This function comes in handy when straight lines on the layout are to
+be examined.
+
+Coordinates outside of the layout bounds are not being considered.
+
+Parameter ``c``:
+    Coordinate whose opposite ones are desired.
+
+Returns:
+    A container that contains pairs of ``c``'s opposing coordinates.)doc";
+
+static const char* __doc_fiction_cartesian_layout_area =
+    R"doc(Returns the layout's number of faces depending on the coordinate type.
+
+Returns:
+    Area of layout.)doc";
+
+static const char* __doc_fiction_cartesian_layout_below =
+    R"doc(Returns the coordinate that is directly below a given coordinate ``c``,
+i.e., the face whose z-dimension is lower by 1. If ``c``'s z-dimension
+is already at minimum, ``c`` is returned instead.
+
+Parameter ``c``:
+    Coordinate whose below counterpart is desired.
+
+Returns:
+    Coordinate directly below ``c``.)doc";
+
+static const char* __doc_fiction_cartesian_layout_cartesian_layout =
+    R"doc(Standard constructor. The given aspect ratio points to the highest
+possible coordinate in the layout. That means in the ASCII layout
+above ``ar = (3,2)``. Consequently, with ``ar = (0,0)``, the layout has
+exactly one coordinate.
+
+Parameter ``ar``:
+    Highest possible position in the layout.)doc";
+
+static const char* __doc_fiction_cartesian_layout_cartesian_layout_2 =
+    R"doc(Copy constructor from another layout's storage.
+
+Parameter ``s``:
+    Storage of another cartesian_layout.)doc";
+
+static const char* __doc_fiction_cartesian_layout_cartesian_layout_storage = R"doc()doc";
+
+static const char* __doc_fiction_cartesian_layout_cartesian_layout_storage_cartesian_layout_storage = R"doc()doc";
+
+static const char* __doc_fiction_cartesian_layout_cartesian_layout_storage_dimension = R"doc()doc";
+
+static const char* __doc_fiction_cartesian_layout_clone =
+    R"doc(Clones the layout returning a deep copy.
+
+Returns:
+    Deep copy of the layout.)doc";
+
+static const char* __doc_fiction_cartesian_layout_coord =
+    R"doc(Creates and returns a coordinate in the layout from the given x-, y-,
+and z-values.
+
+@note This function is equivalent to calling ``OffsetCoordinateType(x,
```

Comment:
@simon1hofmann @marcelwa Do you guys have an idea how to use ``@note`` here? The rendered version does not look as wanted.

Reply:
We probably have to override the docstring in 'documentation.hpp' and use the '..note::' block: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Reply:
Thanks

Reply:
I'm not sure if we should do that or just leave @note in there. 
Otherwise we have to keep the docstrings in documentation.hpp in sync with the original docstrings.

What's your opinion @marcelwa ?

Reply:
`..note::` is an option when it is really important. However, in most cases, we should keep the Python docstrings as easily maintainable as possible. `fiction`'s main focus is still C++. We should try to have Python get in our way as little as possible. I'd say, leave it.

Reply:
Great! That's a good philosophy!